lib/entice/logic/Maps/map.ex

```diff
@@ -50,8 +50,7 @@ defmodule Entice.Logic.Area.Maps do
   end
 end
 
-
-defmodule Entice.Logic.Area.Maps.Map do
+defmodule Entice.Logic.Maps.Map do
```

Comment:
i think we can integrate this into map, since we already generate code for each map in the defmap macro...
and then we can save ourselfes the naming issues with this module :)